themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -57,18 +56,9 @@
 <?php $badgeExclude = ob_get_clean(); ?>
 
 <?php // Render the element: ?>
-<?php if ($hasSubLinks): ?>
-  <span class="<?=implode(' ', $classList) ?>">
-    <a class="text<?=$hasCheckbox ? ' icon-link' : ''?>" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if ($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
-      <?=$displayText ?>
-    </a>
-    <?=$badgeExclude?>
-  </span>
-<?php else: ?>
-  <a href="<?=$toggleUrl ?>" class="<?=implode(' ', $classList) ?><?=$hasCheckbox ? ' icon-link' : ''?>" data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if ($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?> data-lightbox-ignore>
-    <span class="text">
-      <?=$displayText ?>
-    </span>
-    <?=$badgeExclude?>
+<span class="<?=implode(' ', $classList) ?>">
+  <a class="text<?=$hasCheckbox ? ' icon-link' : ''?>" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if ($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
+    <?=$displayText ?>
   </a>
-<?php endif; ?>
+  <?=$badgeExclude?>
```

Comment:
@EreMaijala @demiankatz Was it intentional that the link does not fill the whole `li` element and the badge is not part of it anymore?

The behavior is now different to the account menu (badges for favorite lists) and it does not work that well with our local design.

Reply:
@ThoWagen, are you seeing any visual differences in the core themes, or just in your local customizations? I don't believe there was any intent to change things visually. If the restructuring is having negative side effects, we should at very least add a changelog note about it. @EreMaijala can probably answer better than I can regarding intent/strategy since he did most of the work here. (He's also more likely to be around to discuss further this week since I'm currently on vacation... but I'll try to keep an eye on this in case my help is needed).

Reply:
In the core themes you can not select the facets by clicking on the badge or on the border of the list item anymore. I would expect that to work as a user but maybe only because I'm used to it.

We also change the background color of the list item on hovering in our local theme. But now the background color would change even if nothing can be clicked yet. That might be confusing for users too.

Of course I can adapt to that if the current state was intended.

Reply:
And please enjoy your vacation! I can wait until next week for further responses :)

Reply:
It's intentional, but I'm open to revising it if it's causing problems. The intention was just to improve consistency (and fix some quirks caused by the inconsistency), because we had two different paths depending on whether the exclude option was displayed.

Reply:
I suppose I could change it once more and separate the badge and exclude link from each other.

Reply:
I see what @ThoWagen means about the non-clickable badges. That does feel a little wrong to me as well. I'm not sure what the best solution is; it might be structurally simpler to just put a second link tag around the badge, but that likely would have bad side effects for screen readers.

Reply:
Well, since the badge was part of the link before in case exclude wasn't enabled, it shouldn't be a major issue. I'll check that it doesn't cause any new issues and post a PR if things work out.

Reply:
Thanks @EreMaijala and @demiankatz!